marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/step/manageMappingStep.sjs

```diff
@@ -73,6 +87,11 @@ assertions.push(
 hubJsTest.verifyJson(expectedStep, serviceResponse, assertions);
 hubJsTest.verifyJson(expectedStep, stepService.getStep(stepDefinitionType, stepName), assertions);
 
+assertions.push(
+  test.assertEqual("{}", JSON.stringify(serviceResponse.headers)),
+  test.assertEqual("{}", JSON.stringify(serviceResponse.customHook)),
+  test.assertEqual("[]", JSON.stringify(serviceResponse.processors))
+);
```

Comment:
@srinathgit is this assertion required again here?

Reply:
Yeah, I am calling saveStep() again. So, it ensures the values are present after that as well.